pipelines/cd/event-sourcing-service-delivery.yml

```diff
@@ -0,0 +1,82 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - develop
+  paths:
+    include:
+    - 'HealthcareSystem/EventSourcingService/*'
+pr: none
+
+pool:
+  vmImage: 'ubuntu-18.04'
+
+variables:
+  dockerRegistry: 'pufkedockerhub'
+  project: 'HealthcareSystem/EventSourcingService/EventSourcingService.csproj'
+  dockerfile: 'HealthcareSystem/EventSourcingService/Dockerfile'
+  dockerImage: 'pufke/eventsourcingservice'
+  herokuProject: 'psw-event-store'
+
+stages:
+- stage: CI
+  jobs:
+  - job: Docker
+    steps:
+    - task: Bash@3
+      displayName: 'Prepare context for Docker'
+      inputs:
+        targetType: 'inline'
+        script: |
+          docker --version
+          dotnet publish $(project) -c Release -o publish
+          cp $(dockerfile) publish/Dockerfile  
+    - task: Docker@2
+      displayName: 'Build and push Docker image'
+      inputs:
+        containerRegistry: '$(dockerRegistry)'
+        repository: '$(dockerImage)'
+        command: 'buildAndPush'
+        Dockerfile: 'publish/Dockerfile'
+        buildContext: 'publish'
+        tags: |
+          $(Build.BuildId)
+          latest
+
+- stage: CD
+  variables:
+    HEROKU_API_KEY: 8418d6b4-7423-404b-81cf-23abcd968591
+  jobs:
+  - deployment: EventSourcingService  
+    environment: 'Test'
```

Comment:
We should decide what to do with the test env event database, I would maybe reset it in this pipeline as well just so we don't have issues with missing tables and such if the schema is changed?